### DISCUSSION TITLE
Wyze Color defaults to purple

### DISCUSSION THREAD

**Discussion by toolman11** [SecKatie/ha-wyzeapi#368] [General]
I have about 6 Wyze color bulbs and they always default to purple color in HA with this add-on. They are always set to soft white, but your add-on changes them to purple when accessed through HA. I've tried re-installing the add-on, but it's the same behavior with the latest version. Any ideas? 

![IMG_5241](https://user-images.githubusercontent.com/98511993/162058911-ef7f2dac-d549-4a0e-b517-ecd2ab157fc3.PNG)

**Comment by SecKatie** [comment]
Weird. I am not seeing the same behavior... Can you file an issue and include logs?

**Reply by chrisvball** [reply]
I use the integration and in the past..  the lights would default to a specific color when controlled from HA.  I actually dealt with it this weekend, trying to showoff my Masters tv watching..   anyway, i found that I had a Trigger based rule in the WYZE app, that based on motion from a cam group, trigger certain lights to turn on and "a set color"..   turns out this was the default color.  to fix..  i disable the rule in the App, and then delete the rule from the app..   how into HA and reload the integration.  and give it the normal WYZE slowroll..  90 minutes of so..  good luck
